package.json

```diff
@@ -31,9 +31,9 @@
   },
   "homepage": "https://github.com/digicatapult/dscp-ipfs#readme",
   "dependencies": {
-    "@polkadot/api": "^5.9.1",
+    "@digicatapult/dscp-node": "latest",
     "dotenv": "^8.2.0",
-    "envalid": "^7.0.0",
+    "envalid": "7.0.0",
```

Comment:
there seems to be some bug with default values for `makeValidator` in newer versions of envalid

Reply:
interesting, is there a bug ticket for envalid you've found?

Reply:
reverted